### DISCUSSION TITLE
Please provide a production-ready modular monolithic project template

### DISCUSSION THREAD

**Discussion by l-7-l** [tokio-rs/axum#2152] [Q&A] [answered]
maybe it's overkill, but I guess it's helpful for newbies like me

**Comment by davidpdrsn** [comment, marked as answer]
That is a big ask 😅 Have a look at ECOSYSTEM.md. There are some examples there.

**Comment by Falmarri** [comment]
@l-7-l I'm new to axum _and_ to rust, but I have this you can look at that I've been working on https://github.com/Falmarri/rust-axum-example

I'm sure it has many problems, I was mostly using it as a way to learn rust and axum, but there aren't a lot of server rendered template examples of axum

**Reply by l-7-l** [reply]
Wow, thank you
